src/utilities/mouse_utils.py

```diff
@@ -77,7 +78,7 @@ def click(self) -> bool:
         """
         Clicks on the current mouse position. Identical to pyautogui.click().
         """
-        pag.click()
+        pag.click(interval=(truncated_normal_sample(30, 1500, 133.37, standard_deviation=None)/1000))
```

Comment:
According to [this documentation](https://pyautogui.readthedocs.io/en/latest/mouse.html), the interval parameter is used for the interval between multiple clicks (like double clicking) and not between mouse down and mouse up:
```
To do multiple clicks, pass an integer to the clicks keyword argument. Optionally, you can pass a float or integer to the interval keyword argument to specify the amount of pause between the clicks in seconds. For example:

>>> pyautogui.click(clicks=2)  # double-click the left mouse button
>>> pyautogui.click(clicks=2, interval=0.25)  # double-click the left mouse button, but with a quarter second pause in between clicks
>>> pyautogui.click(button='right', clicks=3, interval=0.25)  ## triple-click the right mouse button with a quarter second pause in between clicks
```

Reply:
I think if you replace it with
```
pag.mouseDown()
time.sleep(truncated_normal_sample(30, 1500, 133.37, standard_deviation=None)/1000) # don't know how correct this is
pag.mouseUp()
```
It will work as expected

Reply:
Yeah, I'm not sure if it's:
click down > delay > click release 
vs  click down > click up (instant) > delay. 

Later I can point it at http://instantclick.io/click-test to see.

Reply:
This example was discussed on Discord by BTWIUSEGENTOO:
```
click_seed = time_seed()
click_down()
time.sleep(normal_seeded(0.030, 0.400, click_seed))
click_up()
```

Reply:
> Yeah, it works but I'm not sure what it's trying to do..
> 
> I'm not sure if it's:
> click down > delay > click release 
> vs  click down > click release (instant) > delay
> 
> Later I can point it at http://instantclick.io/click-test to see.

Doesn't time.sleep() halt execution?
So having the delay between down and up would work?

Reply:
> Doesn't time.sleep() halt execution?
Correct
> So having the delay between down and up would work?
Why wouldn't it?

Reply:
>Doesn't time.sleep() halt execution?
Correct
>So having the delay between down and up would work?
Why wouldn't it? Now it's just a matter of finding the right duration